api/client.go

```diff
@@ -36,6 +37,14 @@ type Config struct {
 	// HttpClient.
 	Address string
 
+	// WrapTTL, if specified, asks the Vault server to return the normal
+	// response wrapped in the cubbyhole of a token, with the TTL of the token
+	// being set to the lesser of this value or a value requested by the
+	// backend originating the response. Specified either as a number of
+	// seconds, or a string duration with a "s", "m", or "h" suffix for
+	// "seconds", "minutes", or "hours" respectively.
+	WrapTTL string
```

Comment:
Seems odd for this to be set at the client level. Seems like an option you would want on a per-request basis, not for the entire client.